src/openforms/contrib/zgw/service.py

```diff
@@ -108,7 +109,7 @@ def create_report_document(
     base64_body = b64encode(submission_report.content.read()).decode()
 
     document_options = {
-        "author": "open-forms",
+        "author": options.get("author", "Aanvrager"),
```

Comment:
this still breaks if someone assigns `options["author"] = ""`

Wouldn't it be better to ensure that `auteur` is set to a non-empty string in the `options` dict _before_ calling these methods?

Reply:
The only place where the client can assign to `author` is in the ZGW API configuration, where "Author" is a required field. How could `options["author"]` get to be equal to an empty string?

Reply:
existing installations already have a working zgw API configuration, so they will typically not touch this, so the admin form validation will not enforce this. When they upgrade, the new column for the CharField is added to the database and it will not have a value (= empty string) since there is no default specified. That's a way that empty strings can end up here.

I missed that this is sourced from the global configuration, so we can fix this by setting a `default="Aanvrager"` on the model field.

However, I still want this intention/expectation to be made explicit, because the zgw config model is pretty distanced from where the value is actually used, so maybe add an `assert options["auteur"], "auteur must be a non-empty string"` to bake this assumption/required into the code.